addons/mass_mailing/static/src/js/mass_mailing_widget.js

```diff
@@ -300,8 +300,6 @@ var MassMailingFieldHtml = FieldHtml.extend({
      */
     _toggleCodeView: function ($codeview) {
         this._super(...arguments);
-        const isFullWidth = !!$(window.top.document).find('.o_mass_mailing_form_full_width')[0];
```

Comment:
Are you sure you didn't break the full-width context then ? I suppose you have tested it but I'm surprised this PR simply replaces slightly convoluted code logic by just a few lines of css. Either we really messed up the code by not considering a css solution earlier, or there is something that was changed in the layout that makes it possible now to achieve the desire behavior using css only ? I'm confused.

Reply:
Yeah honestly I was surprised too, I don't know if something changed in the context that didn't allow us to do this before or if it simply flew above our heads but I tested and didn't notice anything breaking with this.